packages/utilities/fast-web-utilities/src/html.spec.ts

```diff
@@ -4,16 +4,16 @@ import { convertStylePropertyPixelsToNumber, getClientRectWithMargin } from "./h
 describe("getClientRectWithMargin", () => {
     const mockWidth: number = 120;
     const mockHeight: number = 120;
-    const mockRect: ClientRect | DOMRect = {
+    const mockRect: DOMRect = {
         width: mockWidth,
         height: mockHeight,
         top: 0,
         left: 0,
         bottom: 0,
         right: 0,
-        x: undefined,
-        y: undefined,
-        toJSON: undefined,
+        x: undefined!,
+        y: undefined!,
+        toJSON: undefined!,
```

Comment:
Can we drop the changes to `fast-web-utilities` from this PR? I think there is a separate PR of yours out for that. @eisenbergeffect as FYI

Reply:
Yes @chrisdholt , I mistakenly took this branch as base due to which it reflects here too. The link is her for this specific change: 
https://github.com/microsoft/fast/pull/6371